apis/kubedb/constants.go

```diff
@@ -1584,16 +1584,18 @@ const (
 	CassandraVolumeCustomConfig   = "custom-config"
 	CassandraVolumeScript         = "script-volume"
 
-	CassandraVolumeData        = "data"
-	CassandraDataDir           = "/var/lib/cassandra"
-	CassandraServerLogDir      = "var/log/cassandra-server/cassandra-server.log"
-	CassandraServerErrorLogDir = "var/log/cassandra-server/cassandra-server.err.log"
-	CassandraContainerName     = "cassandra"
-	CassandraInitContainerName = "cassandra-init"
-	CassandraMainConfigFile    = "cassandra.yaml"
-	CassandraRackConfigFile    = "rack-config.yaml"
-	CassandraStandalone        = "standalone"
-	CassandraServerConfigFile  = "server-config.yaml"
+	CassandraVolumeData                  = "data"
+	CassandraDataDir                     = "/var/lib/cassandra"
+	CassandraServerLogDir                = "var/log/cassandra-server/cassandra-server.log"
+	CassandraServerErrorLogDir           = "var/log/cassandra-server/cassandra-server.err.log"
```

Comment:
These are not directory.  `File`
May be 3 consts in place of these 3
	CassandraServerDefaultLogDir 	CassandraServerLogFile          CassandraServerErrorLogFile